data/definitions/objects.yml

```
@@ -3969,8 +3969,99 @@ fire_green:
 fire_blue:
   id: 11406
   examine: "Hot!"
+crashed_star_tier_9:
+  id: 38660
+  star: true
+  collect_for_next_layer: 15
+  mining:
+    level: 90
+    life: -1
+    ores: [ stardust ]
+    gems: true
+  examine: "This is a size-9 star"
+crashed_star_tier_8:
+  id: 38661
+  star: true
+  collect_for_next_layer: 40
+  mining:
+    level: 80
+    life: -1
+    ores: [ stardust ]
+    gems: true
+  examine: "This is a size-8 star"
+crashed_star_tier_7:
+  id: 38662
+  star: true
+  collect_for_next_layer: 40
+  mining:
+    level: 70
+    life: -1
+    ores: [ stardust ]
+    gems: true
+  examine: "This is a size-7 star"
+crashed_star_tier_6:
+  id: 38663
+  star: true
+  collect_for_next_layer: 80
+  mining:
+    level: 60
+    life: -1
+    ores: [ stardust ]
+    gems: true
+  examine: "This is a size-6 star"
+crashed_star_tier_5:
+  id: 38664
+  star: true
+  collect_for_next_layer: 175
+  mining:
+    level: 50
+    life: -1
+    ores: [ stardust ]
+    gems: true
+  examine: "This is a size-5 star"
+crashed_star_tier_4:
+  id: 38665
+  star: true
+  collect_for_next_layer: 250
+  mining:
+    level: 40
+    life: -1
+    ores: [ stardust ]
+    gems: true
+  examine: "This is a size-4 star"
+crashed_star_tier_3:
+  id: 38666
+  star: true
+  collect_for_next_layer: 20
+  mining:
+    level: 30
+    life: -1
+    ores: [ stardust ]
+    gems: true
+  examine: "This is a size-3 star"
+crashed_star_tier_2:
+  id: 38667
+  star: true
+  collect_for_next_layer: 10
+  mining:
+    level: 20
+    life: -1
+    ores: [ stardust ]
+    gems: true
+  examine: "This is a size-2 star"
+crashed_star_tier_1:
+  id: 38668
+  star: true
+  collect_for_next_layer: 20
```

```suggestion
  collect_for_next_layer: 450
  mining:
    level: 30
    life: -1
    ores: [ stardust ]
    gems: true
  examine: "This is a size-3 star"
crashed_star_tier_2:
  id: 38667
  star: true
  collect_for_next_layer: 700
  mining:
    level: 20
    life: -1
    ores: [ stardust ]
    gems: true
  examine: "This is a size-2 star"
crashed_star_tier_1:
  id: 38668
  star: true
  collect_for_next_layer: 1200
```


[Wiki 2011](https://runescape.wiki/w/Shooting_Star?oldid=3908155#Star_sizes)
[RS3 Wiki ](https://runescape.wiki/w/Shooting_Star#Mining)

The RS3 wiki number's are only slightly different from 2011, so I'm assuming the 2011 numbers were just slightly off.